### ISSUE TITLE
feelvos, 59.5 on davis17 after my train

### ISSUE THREAD

**Issue by crazyblacker** [tensorflow/models#6792]
Can't achieve 65.9 on davis17 val, just 59.5 after my train. I use a sigle 2080Ti(11G) .

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by saxg** [comment]
I have changed the paremeters except the num_clones=2, train_batch_size=2, as I have 4 gpus. My train set id Youtube Dataset. The training loss keep near 0.9593, and I have dropped down the learning rate to 0.0000001 but the loss is stll near 0.9593.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by juntingzh** [comment]
@crazyblacker Hi, may I know what did you do to improve from 48.3 as in https://github.com/tensorflow/models/issues/6682 to 59.5? When I used the official train.sh with a single GPU, I could only get ~45 val mIOU.
